ONSdigital/eq-survey-runner
# FIX for #1288 upgrade psycopg2 library in requirements

### What is the context of this PR?
500 errors being observed on eq-preprod, see https://github.com/ONSdigital/eq-survey-runner/issues/1288

### How to review 
- launch survey runner in a *venv* with **psycopg2** at version *2.6.1*
- enter some data and then save for later
- then run `pip install -r requirements.txt` and see **psycopg2** using version *2.7.3.1*
- navigate back to survey and data is persisted
- finally submit answers

fixes #1288
